server/src/main/scala/org/dbpedia/extraction/server/resources/TemplateStatistics.scala

```diff
@@ -170,7 +171,7 @@ class   TemplateStatistics(@PathParam("lang") langCode: String, @QueryParam("p")
             var redirectMsg = ""
             for (redirect <- targetRedirect) {
               if (mappingStat.isMapped) {
-                  //redirectMsg = " NOTE: the mapping for " + wikiDecode(redirect, language).substring(createMappingStats.templateNamespace.length()) + " is redundant!"
+                  redirectMsg = " NOTE: the mapping for " + redirect.substring(manager.templateNamespace.length) + " is redundant!"
```

Comment:
I have tried alerts but am not convinced. They use the same color codes already used, so they're either not standing out or confusing. They also add some unwanted padding. Also tried labels but they are a bit too flashy compared to the current tone.